package.json

```diff
@@ -2,24 +2,29 @@
   "scripts": {
     "build": "npm install && cd src/main/webapp && npm install && cd ../../..",
     "dev": "concurrently 'cd src/main/webapp && npm run build:dev' 'mvn package appengine:run'",
+    "test": "cd src/main/webapp && npm run test",
     "prod": "cd src/main/webapp && npm run build:prod && cd ../../../ && mvn package appengine:run",
     "deploy": "cd src/main/webapp && npm run build:prod && cd ../../../ && mvn package war:war appengine:deploy"
   },
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^3.4.0",
     "@typescript-eslint/parser": "^3.4.0",
-    "eslint": "^7.2.0",
+    "eslint": "^7.18.0",
     "eslint-config-google": "^0.14.0",
     "eslint-config-prettier": "^6.10.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.2",
+    "karma": "^6.0.0",
```

Comment:
From what I understood, you shouldn't need karma in this package.json (feel free to test it out and keep it if it breaks your code without it!)

Reply:
I think karma is added by default, karma is in master branch package.json files as well? I tried testing with out and it broke me code, for now I have kept it.